src/interp/interp-trace.cc

```diff
@@ -586,6 +586,26 @@ void Thread::Trace(Stream* stream) {
           ReadU32At(pc + 4), ReadU32At(pc + 8), ReadU32At(pc + 12));
       break;
 
+    case Opcode::V8X16Swizzle:
```

Comment:
This looks to be the same as other v128 binary instructions, combine with the list above (`F64X2Mul`, etc)

Reply:
Done.